main/FatesRestartInterfaceMod.F90

```diff
@@ -1058,7 +1061,7 @@ subroutine set_restart_vectors(this,nc,nsites,sites)
     integer  :: io_idx_co      ! cohort index
     integer  :: io_idx_pa_pft  ! each pft within each patch (pa_pft)
     integer  :: io_idx_pa_cwd  ! each cwd class within each patch (pa_cwd)
-    integer  :: io_idx_pa_sunz ! index for the combined dimensions for radiation
+    integer  :: io_idx_pa_sb   ! each SW band (vis/ir) per patch (pa_sb)
```

Comment:
I think the wavelength band has a terminology (IB?) in CTSM already, so maybe we should mirror that? Altternatively, I think '_SWb' is more understandable than '_sb'.

Reply:
updated naming suffix to _ib